queryfilter/base.py

```diff
@@ -46,6 +46,27 @@ def false_with_drop_none_else_raise(self, field_name):
             return False
         raise FilterOnNoneValue(field_name)
 
+    def get_query_value(self):
+        return self.filter_args["value"]
+
+
+class DjangoQueryFilterMixin(object):
+    __metaclass__ = abc.ABCMeta
+
+    def on_django_query(self, queryset):
+        from django.core.exceptions import FieldError
```

Comment:
Is it good to import here? 🤔

Reply:
if we import locally.   people who are not using django  will not be forced to install django.